sauce/features/budget/check-credit-balances/index.js

```diff
@@ -176,9 +176,7 @@ export class CheckCreditBalances extends Feature {
             display: 'block',
             cursor: 'pointer'
           })
-          .click(function () {
-            this.updateCreditBalances($(this).data('name'), $(this).data('difference'));
-          });
+          .click(this.updateCreditBalances);
```

Comment:
Instead, do not use an anonymous function to call another function, just call that function directly. Here, the "this" keyword is the object CheckCreditBalances.